### PR TITLE
ci: add default_platform to AKS test

### PR DESCRIPTION
**Pull request by burgerdev**

### CODE REVIEW

File: .github/workflows/e2e_aks_runtime.yml [hunk 1]
```diff
@@ -40,6 +41,7 @@ jobs:
           cat <<EOF > justfile.env
           container_registry=${{ env.container_registry }}
           azure_resource_group=${{ env.azure_resource_group }}
+          default_platform=AKS-CLH-SNP
```
Comment (3u13r) [inline .github/workflows/e2e_aks_runtime.yml:44]: We now should be able to remove the platform from the lines 57 and 91

**Review by 3u13r — CHANGES_REQUESTED** [review]

File: .github/workflows/e2e_aks_runtime.yml (hunk 1 above)
Comment (burgerdev) [reply .github/workflows/e2e_aks_runtime.yml:44]: The second one is not overridden by the default, but the first one should be.

**Comment by burgerdev** [comment]
@3u13r: Thanks for debugging this! I changed your commit to use the `PodsFromDeployment` built into contrasttest, that makes it a little more concise.

**Review by 3u13r — APPROVED** [review]
LGTM

**Pull request merged**